frontend/packages/pipelines-plugin/src/components/pipelines/modals/common/PipelineParameterSection.tsx

```diff
@@ -3,11 +3,12 @@ import { FieldArray } from 'formik';
 import { useTranslation } from 'react-i18next';
 import { TextInputTypes } from '@patternfly/react-core';
 import { InputField } from '@console/shared';
-import { PipelineParam } from '../../../../utils/pipeline-augment';
 import FormSection from '@console/dev-console/src/components/import/section/FormSection';
+import { taskParamIsRequired } from '../../pipeline-builder/utils';
+import { ModalParameter } from './types';
 
 type ParametersSectionProps = {
-  parameters: PipelineParam[];
+  parameters: ModalParameter[];
 };
 
 const PipelineParameterSection: React.FC<ParametersSectionProps> = ({ parameters }) => {
```

Comment:
Initial form validation issue is resolved If we introduce this validation hook here.
```suggestion
const PipelineParameterSection: React.FC<ParametersSectionProps> = ({ parameters }) => {
useFormikValidationFix(parameters);

```

Reply:
Ah forgot about that hook! Thanks! Will do that.